### PR TITLE
doc: add grndb --log-flags explanation

### PR DESCRIPTION
**Pull request by kenhys**

### CODE REVIEW

File: doc/source/reference/executables/grndb.rst [hunk 1]
```diff
@@ -118,6 +118,18 @@ Here is an example that specifies ``--log-path`` option::
 
   % grndb check --log-path /var/log/groonga/grndb.log /var/lib/groonga/db/db
 
+``--log-flags``
+"""""""""""""""
+
+.. versionadded:: 9.0.2
+
+It specifies a flag of ``grndb`` log.
```
Comment (komainu8) [inline doc/source/reference/executables/grndb.rst]: In my opinion, we should write about we can output what.

Comment (komainu8) [inline doc/source/reference/executables/grndb.rst]: In my opinion, we should write about the default value.

Comment (kenhys) [reply doc/source/reference/executables/grndb.rst]: Thanks!
I've fixed.

Comment (kenhys) [reply doc/source/reference/executables/grndb.rst]: Thanks!
I've fixed.

**Comment by kenhys** [comment]
@komainu8  I've fixed based on your review, could you review again, please?

**Pull request merged**